notebooks/WFC3/calwf3_v1.0_cte/requirements.txt

```diff
@@ -4,6 +4,6 @@ crds
 jupyter
 matplotlib
 numpy
-photutils
+photutils==1.12.0
 scipy
 wfc3tools
```

Comment:
This directory only has requirements updated, but I see a pre-requirements file that has an older hstcal referenced from astroconda. I think it was just missed

Reply:
@sosey, I reverted this for the reasons mentioned by @FDauphin. Please let me know about the consensus; I can update it accordingly.